osdep/language-posix.c

```diff
@@ -55,8 +60,14 @@ char **mp_get_user_langs(void)
                 continue;
             }
 
+            char *lang = talloc_strndup(ret, envval, len);
```

Comment:
`MP_TARRAY_GROW(NULL, ret, nb);` has to be above this line, else `ret` will be null and this will be potentially leaked string.